### ISSUE TITLE
Towny - Unable to create new towns (NullPointerException)

### ISSUE THREAD

**Issue by Yldales** [nikosgram13/Gringotts#96]
Hi,

**Describe the bug**
Towny isn't able to create a Town when Gringotts is used as economy plugin.

**To Reproduce**
1. Setup a Paper 1.18.2 Minecraft server.
2. Download [Vault .7.3-b131](https://www.spigotmc.org/resources/vault.34315/download?version=344916).
3. Download Gringotts 2.12.3.
4. Download [Towny 0.98.2.2](https://github.com/TownyAdvanced/Towny/releases/download/0.98.2.2/towny-0.98.2.2.jar).
5. As a player in-game, execute `/town create Test`.

**Expected behavior**
The Towny's town "Test" should be created.

**Screenshots**
![image](https://user-images.githubusercontent.com/50544935/172433738-47dcce5a-3089-44df-904e-3f7ab1b4cfbb.png)

**Desktop (please complete the following information):**
 - **Server Type** : Paper
 - **Server Version** : [Paper 379 (MC: 1.18.2) (Implementing API version 1.18.2-R0.1-SNAPSHOT) (Git: 276d830)](https://api.papermc.io/v2/projects/paper/versions/1.18.2/builds/379/downloads/paper-1.18.2-379.jar)
 - **Gringotts Version** : 2.12.3 (compiled from ad56547665a4b3697adf8d076a5ab1da31d61d9c)

**Additional context**
- Error log : https://pastebin.com/E1aCq0iz
- Vault, Gringotts and Towny are using their default configuration.

**Comment by PainOchoco** [comment]
I have the same issue. I tried with version of Gringotts 2.12.1 and town creation seems to be working ^^
I'm gonna try to track where is the issue between both versions

**Comment by Yldales** [comment]
> I have the same issue. I tried with version of Gringotts 2.12.1 and town creation seems to be working ^^ I'm gonna try to track where is the issue between both versions

Oh, you're right. 
The problem seems to be with the 8eea3fb commit. 
The problem is not present in the previous commit (ae0eac3).

**Comment by PainOchoco** [comment]
Yes, seems like it's from [TownHolderProvider.java](https://github.com/nikosgram/gringotts/commit/8eea3fb2ee1673ba0a50397f5ab23affb678d0ee#diff-0ac9611aea4b21176c70fa1204b5b28f68de366e7418a595e92ec3f359a4ce6d) line 54.

**Comment by PainOchoco** [comment]
I rollbacked the changes and built the plugin and it works fine!

Here are the changes that I rollbacked:
- [NationHolderProvider.java](https://github.com/nikosgram/gringotts/commit/8eea3fb2ee1673ba0a50397f5ab23affb678d0ee#diff-65ecb935493a5a63fb3680644cb128512d7a6d283b75702f552153ef4c8c015e) l.52 -> l.64; l.68; l.79 -> l.84
- [TownHolderProvider.java](https://github.com/nikosgram/gringotts/commit/8eea3fb2ee1673ba0a50397f5ab23affb678d0ee#diff-0ac9611aea4b21176c70fa1204b5b28f68de366e7418a595e92ec3f359a4ce6d) l.51 -> l.63; l.67; l.78 -> l.83.

The previous code uses deprecated method from Towny but it should be fixable.
